config/initializers/inflections.rb

```diff
@@ -15,6 +14,7 @@
   inflect.acronym 'GI'
   inflect.acronym 'IHub'
   inflect.acronym 'MDOT'
+  inflect.acronym 'NCA' # National Cemetery Administration
```

Comment:
This literally doesn't need any filename/locations changed? Or a 'require' removed?

Reply:
no it only occurs in the `app/models/facilities/nca_facility` file, and it won't load when zeitwerk is turned on unless we rename the class to `NcaFacility`